src/kOS/Module/kOSProcessor.cs

```diff
@@ -132,6 +130,26 @@ public Screen.TermWindow GetWindow()
 
         public override void OnStart(StartState state)
         {
+			//Populate selector for boot scripts
```

Comment:
I'll add a check for StartSate and move all the logic into a separate Method for a cleaner look.